WolvenKit.App/ViewModels/Shell/ChunkViewModel.cs

```diff
@@ -1229,10 +1229,10 @@ private void DeleteItem()
 
     private bool CanImportWorldNodeData() => Data is worldNodeData && PropertyCount > 0;   // TODO RelayCommand check notify
     [RelayCommand(CanExecute = nameof(CanImportWorldNodeData))]
-    private Task ImportWorldNodeDataTask() => ImportWorldNodeDataTask(true);
+    private Task ImportWorldNodeDataAsync() => ImportWorldNodeDataTask(true);
```

Comment:
Does this not need to be
`private async Task ImportWorldNodeDataAsync() => await ImportWorldNodeDataTask(true);`